examples/api-tests/src/file-search.spec.js

```diff
@@ -55,6 +57,59 @@ describe('file-search', function () {
 
         });
 
+        describe('#buildOpenerOptions', () => {
```

Comment:
Take a look at the formatting of `describe('#compareItems', () => {` we should:
- add a `newline` after `describe` statements
- add a `newline` in between `it` statements